media-api/app/lib/querysyntax/QuerySyntax.scala

```diff
@@ -37,6 +37,7 @@ class QuerySyntax(val input: ParserInput) extends Parser with ImageFields {
     "copyright" |
     "source" |
     "category" |
+    "preset" |
```

Comment:
Preset is really a horrible picdarism, why not category (though that clashes with the current usage rights categories, so maybe not) or, to polyfill the future, subject maybe?